src/pages/search.tsx

```diff
@@ -32,10 +33,19 @@ export default function PageGatsbySearch(props: PageProps): ReactElement {
       }`
 
   return (
-    <Page title={title} uri={props.uri}>
+    <Page
+      title={title}
+      description={
+        totalPagesNumber &&
+        totalPagesNumber > 476 &&
+        'More than 10k results were found, displaying the first 10k. Please refine your search'
```

Comment:
In general, it works very nicely and is a good solution. I think we could refine the message slightly as it says "60876 results for data" and then "More than 10k results were found" which seems like repetition. 

![Pagination message](https://user-images.githubusercontent.com/20739535/131833771-273fad27-bb49-4e5a-8e37-da9a3d0959ae.png)

I would suggest a shorter message "Results are limited to the first 10k, please refine your search."

```suggestion
        'Results are limited to the first 10k, please refine your search.'
```

Reply:
Thanks for the feedback. It might sound a bit repetitive and also not sure if that number '60876'  of results makes any sense in this context.
So the title could be `More than 10k results for data` and in description 'First 10k results are displayed. Please refine your search' or simply 'Please refine your search' for description

Reply:
agree, if we only display 10000 results then the title should reflect that, maybe `>10000 results for data`. As for the message I'm afraid people will miss it as their eyes most likely will jump to asset list results right away. Not sure how feasible it is to put an `<Alert state="info">` into that `description`, if we markdown-process that already we could at least make it bold with `**Results displayed are limited to the first 10k, please refine your search.**`

Reply:
showing an `alert` components doesn't seem too feasible without modifying some components because right now it waits for the description as a string. I don't think it's worth modifying it for this scope, because in most of the cases the user will refine the search when he sees this much results.